.github/workflows/fast_testing.yml

```diff
@@ -10,6 +10,40 @@ on:
       - '*'
 
 jobs:
+  version_check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v3
+
+      - name: Setup tarantool
+        uses: tarantool/setup-tarantool@v2
+        with:
+          tarantool-version: '2.10'
+
+      - name: Package version
+        # Found that Github checkout Actions pulls all the tags, but
+        # right it deannotates the testing tag, check:
+        #   https://github.com/actions/checkout/issues/290
+        # But we use 'git describe ..' calls w/o '--tags' flag and it
+        # prevents us from getting the needed tag for packages version
+        # setup. To avoid of it, let's fetch it manually, to be sure
+        # that all tags will exists always. The actions/checkout depth
+        # does not help.
+        run: |
+          git fetch --tags -f
+          tarantoolctl rocks make
+          VERSION=$(tarantool -e "print(require('queue')._VERSION)")
+          REPO_LAST_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
+          if [ "$VERSION" != "$REPO_LAST_TAG" ]; then
+            echo VERSION from code is $VERSION
```

Comment:
I think you may extract these two lines from `if fi` block since they are useful.

Reply:
Done